xarray/core/ops.py

```diff
@@ -325,24 +331,35 @@ def inject_all_ops_and_reduce_methods(cls, priority=50, array_only=True):
 
     # patch in standard special operations
     for name in UNARY_OPS:
+        if hasattr(cls, op_str(name)):
+            name = "_injected_" + name
         setattr(cls, op_str(name), cls._unary_op(get_op(name)))
     inject_binary_ops(cls, inplace=True)
 
     # patch in numpy/pandas methods
     for name in NUMPY_UNARY_METHODS:
+        if hasattr(cls, op_str(name)):
+            name = "_injected_" + name
         setattr(cls, name, cls._unary_op(_method_wrapper(name)))
 
     for name in PANDAS_UNARY_FUNCTIONS:
         f = _func_slash_method_wrapper(getattr(duck_array_ops, name), name=name)
+        if hasattr(cls, op_str(name)):
+            name = "_injected_" + name
         setattr(cls, name, cls._unary_op(f))
 
     f = _func_slash_method_wrapper(duck_array_ops.around, name="round")
-    setattr(cls, "round", cls._unary_op(f))
+    if hasattr(cls, "round"):
+        setattr(cls, "_injected_round", cls._unary_op(f))
```

Comment:
The idea was that `round` should behave the same as all the other injected methods, in that if `round` was (at some point in the future) explicitly defined as a method of say `DataArray`, it would override the currently used `duck_array_ops.around`. This is what would happen for all the other injected methods, so I thought it would be more consistent for `round` to behave the same way, rather than be skipped because it's already treated specially here. Happy to revert this change if you prefer.